applications/StructuralMechanicsApplication/custom_constitutive/generic_finite_strain_isotropic_plasticity.cpp

```diff
@@ -45,237 +45,313 @@
 namespace Kratos
 {
 template<class TElasticBehaviourLaw, class TConstLawIntegratorType>
-void GenericFiniteStrainIsotropicPlasticity<TElasticBehaviourLaw, TConstLawIntegratorType>::CalculateMaterialResponseKirchhoff(ConstitutiveLaw::Parameters& rValues)
+void GenericFiniteStrainIsotropicPlasticity<TElasticBehaviourLaw, TConstLawIntegratorType>::CalculateMaterialResponsePK1(ConstitutiveLaw::Parameters& rValues)

```

Comment:
has to be kirchoff because souza says?

Reply:
I just reordered to be consistent with the header. But we compute Kirchof because the predictor (hyper elastic law) computes kirchoff